src/Utils/Data/ExchangeRateFetcher.java

```diff
@@ -68,19 +67,14 @@ public static long getLastFetchTime() {
         return lastEndMillis - lastStartMillis;
     }
 
-    public static String getLastFetchTimeAsString() {
-        String fetchTimeAsString = String.valueOf(getLastFetchTime);
-        return fetchTimeAsString;
-    }
-
     // Gibt den zu letzt gespeicherten Wechselkurs zurück
     public static double getLatestExchangeRate() {
         return latestRate;
     }
 
     /*
      * Diese Methode setzt bei einer Fehlermeldung die
-     * Daten zurück damit keine Nullen ausgegeben werden
+     * Daten zurück damit keine "Null" ausgegeben werden
```

Comment:
In Java gibt es "null" und ich glaub das könnte echt verwirren. Können wir einfach 0 schreiben oder das schreiben was ich da vorher geschrieben hab?